vignettes/getting-started-part-I.Rmd

```diff
@@ -0,0 +1,220 @@
+---
+title: "Getting Started with Epinowcast"
```

Comment:
Should we call this getting started nowcasting with epinowcast or similar  (to foreshadow other getting started options that may not be for serial consumption?

Reply:
Very sensible, will update.